src/main/java/com/epam/trn/hw6/dataProvider/JdiDataProvider.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.trn.hw6.dataProvider;
+
+import com.epam.trn.hw6.utils.JsonDataReader;
+import org.testng.annotations.DataProvider;
+
+import java.io.IOException;
+
+public class JdiDataProvider {
+
+    protected JsonDataReader reader = new JsonDataReader();
+
+    public JdiDataProvider() throws IOException {
+    }
+
+    @DataProvider
+    public Object[][] data() {
+        return new Object[][] {
+                {reader.getDataSet("data_1")},
```

Comment:
This is completely prohibited to create code with reference to amount of data. What if we have to increase data set ?